### ISSUE TITLE
Inconsistency in parameter naming between dnvm alias and other dnvm commands

### ISSUE THREAD

**Issue by moozzyk** [aspnet/dnvm#311]
`dnvm alias` uses only positional parameters: 

```
dnvm alias
  Lists and manages aliases

usage:
  dnvm alias [-d] [<Name>] [<Version>] [<Architecture>] [<Runtime>]
```

other commands (e.g. `dnvm use`, `dnvm install`) use `-arch` for architecture and '-r` for runtime

Similarly for `dnvm install` uses `-a` for alias while `dnvm upgrade` or `dnvm use` use it as positional parameter

**Comment by tshak** [comment]
:+1:

**Comment by dougbu** [comment]
@moozzyk though the help may still be imperfect, looks like the commands are now consistent. #452 and #455 address a few specific issues. But otherwise is it time to close this issue?

On your specific points:
> `dnvm alias` uses only positional parameters

`dnvm alias` takes parameters such as `-arch {architecture}` and `-runtime {runtime}` on both Windows and non-Windows platforms. I believe the main remaining issue in this area is #400 and that's Windows-specific.

> `dnvm install` uses `-a` for alias while `dnvm upgrade` or `dnvm use` use it as positional parameter

Strictly speaking `dnvm install` accepts an alias in two places. The first matches how an alias may be used in `dnvm upgrade` or `dnvm use` e.g. `dnvm install default -arch x64`. The second, non-positional parameter _creates_ an alias.

That said `-a` won't be an alias for `-arch` (or `-architecture` on Windows) once #455 is done.

**Comment by moozzyk** [comment]
Yes. This seems to work fine. #455 takes care of the issue where the same switch has different meaning depending on the command. Help could be improved (especially on Linux where we don't show that -r, -arch etc. are valid for this command). 

I see one discrepancy between Windows and Linux where on Windows you can create an alias for a runtime that is not installed whereas on Linux it returns an error saying that the version is not installed. 

Also Windows version have -OS switch - I don't know if this is supposed to be supported on Linux. Currently it seems to be a no-op. Which may be fine if we, in general, don't fail on unrecognized parameters.

The last two are not part of this issue though. So I think this one can be closed - and if the things I brought up are not by design we should create separate issues to track them.
